CHANGELOG-API.md

```diff
@@ -7,6 +7,10 @@ If you are looking for user-facing changes, check out [CHANGELOG.md](./CHANGELOG
 
 <!-- next version -->
 
+## v0.102.1
+
+No API-only changes on this release.
```

Comment:
Should we call out the CVE here anyways as an attempt to tangentially inform people who look at this changelog?

Reply:
Added some louder text